hathor/indexes/manager.py

```diff
@@ -184,27 +173,20 @@ def _manually_initialize(self, tx_storage: 'TransactionStorage') -> None:
         for index in self.iter_all_indexes():
             index.init_start(self)
 
-        self.log.debug('indexes init')
         if indexes_to_init:
-            tx_iter = progress(tx_storage.topological_iterator(), log=self.log, total=tx_storage.get_vertices_count())
+            overall_scope = reduce(operator.__or__, map(lambda i: i.get_scope(), indexes_to_init))
```

Comment:
I think it's a bit more readable if you don't use that extra `map`, and also by doing this you get the advantage of using the `__or__` operator that was implemented:

```suggestion
            overall_scope = reduce(operator.__or__, map(lambda i: i.get_scope(), indexes_to_init))
            overall_scope = reduce(lambda i1, i2: i1.get_scope() or i2.get_scope(), indexes_to_init)
```

What do you think?

Reply:
Although it would be more readable, it doesn't quite work, `lambda i1, i2: i1.get_scope() or i2.get_scope()` takes two indexes and returns one scope, but reduce works by chaining the result of the previous call as first argument of the next call, so `s0 = fun(i[0], i[1])` and then `s1 = fun(s0, i[2])` would break it.

The `or` operator is still being used `operator.__or__` is just the function version to use with functional stuff, so `a or b` is equivalent to `operator.__or__(a, b)`. But, yeah, it's less readable.

Reply:
That's true! Sorry, my mistake. Then maybe:
```suggestion
            all_scopes = map(BaseIndex.get_scope, indexes_to_init)
            overall_scope = reduce(operator.__or__, all_scopes)
```